emoji-contest-25.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: fr
+title: emoji-contest-25
+author: dismad
+date: January 12, 2025
+amount: 5000 VTC
+milestones:
+  - name: Emoji Contest Community Fund
+    funds: 100%
+    done:
+    status: unfinished
+payouts:
+  - date:
+    amount:
+  - date:
+    amount:
+---
+
+
+
+
+# Emoji-contest-25
+
+```
+* Welcome to the 5th annual Vertcoin emoji contest 
```

Comment:
Specify that emoji contest is for Vertcoin Discord and provide link to server in introduction